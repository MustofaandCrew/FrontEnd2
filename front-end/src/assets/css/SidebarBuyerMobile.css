.foto-profil {
    width: 35px;
    border-radius: 20px;
    margin-right: 10px;
}
.logo-sidebar-mobile {
    height: 60px;
    width: 50px;
    margin-right: 20px;
}
.item-mobile {
    margin-top: 20px;
    margin-left: 20px;
}