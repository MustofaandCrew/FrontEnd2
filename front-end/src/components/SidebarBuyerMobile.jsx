import React from "react";
import Logo from "../assets/images/logo.png";
import "../assets/css/SidebarBuyerMobile.css";

export default function SideBuyerMobile(props) {
  return (
    <nav className="navbar bg-light fixed-top">
      <div className="container-fluid">
        <button className="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar">
          <span className="navbar-toggler-icon"></span>
        </button>
        <div className="offcanvas offcanvas-start" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
          <div className="offcanvas-header">
            <h5 className="offcanvas-title" id="offcanvasNavbarLabel">
              <a href="/">
                <img src={Logo} className="logo-sidebar-mobile" />
              </a>
              SecondHand
            </h5>
            <button type="button" className="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
          </div>
          <div className="offcanvas-body">
            <ul className="navbar-nav justify-content-end flex-grow-1 pe-3">
              <li className="nav-item item-mobile text">
                <a className="nav-link active" aria-current="page" href="/transaksibuyer">
                  Dashboard
                </a>
              </li>
              <li className="nav-item item-mobile text">
                <a className="nav-link active" href="/accountbuyer">
                  My Account
                </a>
              </li>
            </ul>
          </div>
        </div>
        <a className="navbar-brand" href="#">
          <img src={props.image} alt="profil" className="foto-profil" />
          Hi, {props.nama}
        </a>
      </div>
    </nav>
  );
}
