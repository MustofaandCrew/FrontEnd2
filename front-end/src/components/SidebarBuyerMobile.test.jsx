import React from "react";
import renderer from "react-test-renderer";
import SidebarBuyerMobile from "../components/SidebarBuyerMobile";


describe("Test suits for SidebarBuyerMobile", () => {
    it("should match with snapshot", () => {
        const tree = renderer.create(<SidebarBuyerMobile />).toJSON();
    });
});