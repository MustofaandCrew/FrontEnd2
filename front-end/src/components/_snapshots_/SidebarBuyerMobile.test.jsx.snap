// Jest Snapshot v1, https://goo.gl/fbAQLP
import SidebarBuyerMobile from "../components/SidebarBuyerMobile";

exports["SidebarBuyerMobile"] = {
    test: function test() {
        var tree = renderer.create(<SidebarBuyerMobile />).toJSON();
        expect(tree).toMatchSnapshot();
    }
};